3Box/3box-profile-hover.md

```diff
@@ -0,0 +1,210 @@
+
+### Summary
+
+
+One of the biggest frictions with onboarding users to Web3 is overcoming a lot of the technical issues that are required by design from decentralized networks and one of them are public addresses which are represented by a 40 character hexadecimal hash which is very unappealing for users.
```

Comment:
```suggestion
One of the biggest frictions with onboarding users to Web3 is overcoming a lot of the technical issues with decentralized networks and one of them are public addresses which are represented by a 40 character hexadecimal hash which is unappealing for users.
```

Suggest changing this a bit, as I don't think the issues are required, but just part of it right now.